apps/web-giddh/src/app/settings/linked-accounts/setting.linked.accounts.component.ts

```diff
@@ -21,7 +21,7 @@ import { GIDDH_DATE_FORMAT } from '../../shared/helpers/defaultDateFormat';
     selector: 'setting-linked-accounts',
     templateUrl: './setting.linked.accounts.component.html',
     styles: [`
-    .bank_delete {
+    .bank-delete {
```

Comment:
make its separate scss file

Reply:
it has single code of css

Reply:
@aditi-tiwari17 please confirm with @prateek-giddh or @ravindersinghdang

Reply:
@prateek-giddh it has single line css, please confirm if separate file for css is required in this

Reply:
done